jerry-core/ecma/base/ecma-helpers-string.c

```diff
@@ -1263,6 +1263,24 @@ ecma_string_get_length (const ecma_string_t *string_p) /**< ecma-string */
 } /* ecma_string_get_length */
 
 /**
+ * Checks whether is ecma string empty or not
+ *
+ * @return true  - if empty
+ *         false - otherwise
+ */
+bool
```

Comment:
Is this change for binary size reduction only or is performance improvement also a goal? If so, could this function not be a candidate for inlining? (`inline` an `__attr_always_inline___`) Or would that blow up the code?